draft-ietf-ppm-dap.md

```diff
@@ -2449,62 +2389,23 @@ In the presence of this adversary, DAP aims to achieve the privacy and
-Details for each issue are below.
-
-1. When crafted maliciously, collect requests may leak more information about
-   the measurements than the system intends. For example, the spec currently
-   allows sequences of collect requests to reveal an aggregate result for a
-   batch smaller than the minimum batch size. [OPEN ISSUE: See issue#195. This
-   also has implications for how we solve issue#183.]
 1. Even benign collect requests may leak information beyond what one might
-   expect intuitively. For example, the Poplar1 VDAF
-   {{!VDAF}} can be used to compute the set of heavy hitters among a set of
-   arbitrary bit strings uploaded by Clients. This requires multiple evaluations
-   of the VDAF, the results of which reveal information to the Aggregators and
-   Collector beyond what follows from the heavy hitters themselves. Note that
-   this leakage can be mitigated using differential privacy.
-   [OPEN ISSUE: We have yet not specified how to add DP.]
-1. The core DAP spec does not defend against Sybil attacks. In this type of
-   attack, the adversary adds to a batch a number of reports that skew the
-   aggregate result in its favor. For example: The result may reveal additional
-   information about the honest measurements, leading to a privacy violation; or
-   the result may have some property that is desirable to the adversary ("stats
-   poisoning"). The upload sub-protocol includes an extensions mechanism that
-   can be used to prevent --- or at least mitigate --- these types of attacks.
-   See {{upload-extensions}}. [OPEN ISSUE: No such extension has been
-   implemented, so we're not yet sure if the current mechanism is sufficient.]
-1. The choice of VDAF can impact the computation required for a DAP Task. For
-   instance, the Poplar1 VDAF {{!VDAF}} when configured to compute a set of
-   heavy hitters requires each measurement to be of the same bit-length which
-   all parties need to agree on prior to VDAF execution. The computation
-   required for such tasks can increase superlinearly as multiple rounds of
-   evaluation are needed for each bit of the measurement value.
-
-   When dealing with variable length inputs (e.g domain names), it is
-   necessary to pad them to convert into fixed-size measurements. When
-   computing the heavy hitters from a batch of such measurements, we can
-   early-abort the Poplar1 execution once we have reached the padding region
-   for a candidate measurement. For smaller length inputs, this significantly
-   reduces the cost of communication between Aggregators and the steps
-   required for the computation. However, malicious Clients can still generate
-   maximum length inputs forcing the system to always operate at worst-case
-   performance.
-
-   Therefore, care must be taken that a DAP deployment can comfortably handle
-   computation of measurements for arbitrarily large sizes, otherwise, it may
-   result in a DoS possibility for the entire system.
```

Comment:
This discussion is now down in "Task parameters", below, to keep this list focused on problems that DAP doesn't address